pkgs/development/tools/zigmod/buildZigmodPackage.nix

```diff
@@ -0,0 +1,78 @@
+{ lib
+, stdenv
+, cacert
+, git
+, zig
+, zigmod
+}:
+
+{ name ? "${attrs.pname}-${attrs.version}"
```

Comment:
I am not sure if we still do this for new things since we always want plane+version